plugin/pkg/admission/resourcequota/controller.go

```diff
@@ -166,7 +173,9 @@ func (e *quotaEvaluator) checkAttributes(ns string, admissionAttributes []*admis
 		}
 		return
 	}
-	if len(quotas) == 0 {
+	// if limited resources are disabled, we can just return safely when there are no quotas.
+	limitedResourcesDisabled := len(e.config.LimitedResources) == 0
```

Comment:
if `e.config` must be non-nil, why do you allow a nil to be passed through the `New` function?